clients/python/game/api.py

```diff
@@ -0,0 +1,27 @@
+from requests import get
+
+from .models import Game
+
```

Comment:
[PEP8 convention](https://www.python.org/dev/peps/pep-0008/#blank-lines), which is why I tend to do it by mistake in my Scala commits. :)